src/utils/mmap_backend.rs

```diff
@@ -85,10 +85,17 @@ impl<W> MmapBackend<W> {
             .len()
             .try_into()
             .with_context(|| "Cannot convert file length to usize")?;
-        let file = std::fs::File::open(path.as_ref())
-            .with_context(|| "Cannot open file for MmapBackend")?;
         // Align to multiple of size_of::<W>
         let mmap_len = file_len.align_to(size_of::<W>());
+        #[cfg(windows)]
+        {
+            ensure!(
+                mmap_len == file_len,
+                "Cannot mmap a non-aligned file in read-only mode. Use \"webgraph.exe pad BASENAME\" to ensure alignment in graph files."
```

Comment:
Nope, this is more like "File has insufficient padding for read word size {}; please use the pad command to ensure sufficient padding", W::BITS. Note that `.exe` is a Windows-only thing and frowned upon.

Reply:
Should I leave the `.exe` out even if this can only occur on Windows (as this isn't even compiled for other platforms)?